### DISCUSSION TITLE
Get the latest news about EJBCA Community and other open-source software

### DISCUSSION THREAD

**Discussion by KarolinHem** [Keyfactor/ejbca-ce#679] [Announcements]
The Keyfactor Community newsletter offers the latest product updates and resources from our open-source software EJBCA Community, SignServer Community, and Bouncy Castle. 

Stay up to date on the latest news, software updates, hands-on tutorials, and user workshops on Keyfactor’s open-source software. 

**[Sign up for the newsletter](https://www.ejbca.org/#popup-newsletter)**

**Comment by NeerajGahlawatJava** [comment]
Hi Team,

I am using EJBCA Community Edition to create certificates and have deployed EJBCA using Helm, where TLS_SETUP_ENABLED="simple".

As part of our automation script, we access the EJBCA Admin Web page to initiate the configuration. However, we are encountering an issue where a TLS setup popup (screenshot attached) appears, interrupting the automation process.

![Screenshot 2025-03-05 133648](https://github.com/user-attachments/assets/dd7403f6-0612-4e2e-9199-d19e41ad3b08)

We need a way to bypass or disable this TLS popup so that our automation script can proceed smoothly with the EJBCA configuration.

Could you please suggest an approach to prevent the TLS popup from appearing when accessing the EJBCA Admin Web page?

Thanks in advance!


Regards,
Neeraj Gahlawat
